action_server/src/action_server/actions/action.py

```diff
@@ -107,16 +107,19 @@ def _configure(self, robot, config):
         raise NotImplementedError
 
     def start(self):
-        rospy.loginfo("Starting executing of action {}.".format(self.__class__.__name__))
+        rospy.loginfo("Starting executing of action {}.".format(self.get_name()))
         self._start()
         return self._execute_result
 
     def _start(self):
         raise NotImplementedError
 
     def cancel(self):
-        rospy.loginfo("Canceling executing of action {}.".format(self.__class__.__name__))
+        rospy.loginfo("Canceling executing of action {}.".format(self.get_name()))
         return self._cancel()
 
     def _cancel(self):
         raise NotImplementedError
+
+    def get_name(self):
+        return self.__class__.__name__
```

Comment:
Using `type(self)` instead of `self.__class__` you would access a private attribute one less time. Still need the `__name__` though so not that important.